sanic_routing/route.py

```diff
@@ -108,25 +133,15 @@ def _setup_params(self):
             parts = path_to_parts(key_path, self.router.delimiter)
             for idx, part in enumerate(parts):
                 if "<" in part:
-                    if ":" in part:
-                        (
-                            name,
-                            label,
-                            _type,
-                            pattern,
-                        ) = self.parse_parameter_string(part[1:-1])
-                        self.add_parameter(
-                            idx, name, key_path, label, _type, pattern
-                        )
-                    else:
-                        self.add_parameter(
-                            idx,
-                            part[1:-1],
-                            key_path,
-                            "string",
-                            str,
-                            self.router.regex_types["string"],
-                        )
+                    (
+                        name,
+                        label,
+                        _type,
+                        pattern,
+                    ) = self.parse_parameter_string(part[1:-1])
```

Comment:
Horrible formatting. Could this be kept to a single line? (doesn't seem to be too far above the column limit).

Reply:
Black is persistent. We defer to `black`.